tests/src/test/java/alluxio/server/tieredstore/SpecificTierWriteIntegrationTest.java

```diff
@@ -45,8 +45,8 @@
  * Integration tests for writing to various storage tiers.
  */
 public class SpecificTierWriteIntegrationTest extends BaseIntegrationTest {
-  private static final int CAPACITY_BYTES = Constants.KB;
-  private static final int FILE_SIZE = CAPACITY_BYTES;
+  private static final int CAPACITY_BYTES = Constants.KB * 10;
```

Comment:
Why are we bumping these?

Reply:
When file size = capacity bytes, it is difficult to test eviction (all or nothing). This also isn't the expected case in normal usages of Alluxio, we expect each worker to be able to store more than one file/block.